<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomKiosk</class>
 <widget class="QMainWindow" name="RoomKiosk">
  <property name="windowTitle">
   <string>회의실 키오스크</string>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>380</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_root" stretch="0,1,0">
    <!-- 상단: 연결/상태 -->
    <item>
     <layout class="QHBoxLayout" name="layout_top">
      <item>
       <widget class="QLabel" name="lbl_status">
        <property name="text">
         <string>상태: 준비 중</string>
        </property>
        <property name="minimumWidth">
         <number>280</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="spacer_top">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_status">
        <property name="text">
         <string>상태조회 (HR)</string>
        </property>
        <property name="toolTip">
         <string>아두이노로 HR 전송</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>

    <!-- 중간: 인증번호 + 입실/퇴실 -->
    <item>
     <layout class="QGridLayout" name="grid_auth">
      <item row="0" column="0">
       <widget class="QLabel" name="label_code">
        <property name="text">
         <string>인증번호</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_code">
        <property name="placeholderText">
         <string>4자리</string>
        </property>
        <property name="inputMask">
         <string>9999;_</string>
        </property>
        <property name="maxLength">
         <number>4</number>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btn_enter">
        <property name="text">
         <string>입실</string>
        </property>
        <property name="toolTip">
         <string>인증번호 확인 후 EN 1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="btn_leave">
        <property name="text">
         <string>퇴실</string>
        </property>
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>입실 후 인증번호 재입력 → EN 0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>

    <!-- 하단: 메시지 영역 -->
    <item>
     <widget class="QLabel" name="lbl_msg">
      <property name="text">
       <string>안내 메시지가 여기에 표시됩니다.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="minimumHeight">
       <number>140</number>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
